cargo-cyclonedx/src/config.rs

```diff
@@ -85,6 +88,29 @@ impl FromStr for IncludedDependencies {
     }
 }
 
+#[derive(Debug, Clone, Default, PartialEq, Eq, Deserialize)]
+pub struct LicenseParserOptions {
+    /// Use lax or strict parsing
+    #[serde(default)]
+    pub strict: bool,
+
+    /// Silently accept the named licenses
+    #[serde(default)]
+    pub accept_named: HashSet<String>,
+}
+
+impl LicenseParserOptions {
+    pub fn merge(mut self, other: Self) -> Self {
+        Self {
+            strict: other.strict,
+            accept_named: {
+                self.accept_named.extend(other.accept_named);
+                self.accept_named
+            },
+        }
+    }
+}
```

Comment:
I amended most of the points (not pushed yet).

I am not so sure about the `accept_named` field. The idea was to allow the re-use of workspace configuration. Assuming you have a bunch of licenses you just want to allow as a named license (not spdx) then repeating those in the project configuration for every project might be a frustrating experience. So I made them additive, which still allows one to have distinct project configurations (by not configuring anything on the workspace level), but also allows one overarching workspace configuration, which doesn't get overridden by the project configuration in case it is present.

True, for the CLI this means right now that it cannot be overridden.

I don't have a strong opinion on this one. It just felt like a corner case, which could get really frustrating if you have a lot of projects with custom license names. Just let me know if I should still refactor this as suggested.